osmosis-1/osmosis-1.assetlist.json

```diff
@@ -11242,7 +11242,7 @@
         "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sge/images/sge.png",
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/sge/images/sge.svg"
       },
-      "coingecko_id": "sge",
+      "coingecko_id": "six-sigma",
```

Comment:
Please make this change at the Chain Registry.

Reply:
Already did.

Reply:
Oh, sorry, you indeed already had.

Reply:
No worries @JeremyParish69 .Thanks